images/scripts/arch.setup

```diff
@@ -12,6 +12,7 @@ export LC_ALL=C
 COCKPIT_DEPS="\
 binutils \
 edk2-ovmf \
+gsettings-desktop-schemas \
```

Comment:
This is highly dubious. This is totally *not* a cockpit dependency. Please report a bug against arch about this (libosinfo has a CLI which should reproduce this). libosinfo should need a desktop package, and if it really really has to, it needs to declare a dependency by itself. After that, please move this to TEST_PACKAGES with a `# HACK` comment. Thanks!